### PR TITLE
Reload tree on Enter key pressed

### PR DESCRIPTION
**Pull request by hansnet**

### CODE REVIEW

File: src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor [hunk 1]
```diff
@@ -55,7 +55,7 @@
                 <Badge Pill="true" Color="Color.Danger" style="top:-35px;left:8px;height:14px;font-size:0.6em">@(this.NewItemsCount)</Badge>
             }
         </button>
-        <input class="form-control form-control-sm ml-1 rounded " style="padding-right:33px" id="AvatarUrl" placeholder="Avatar URL" @bind="AvatarUrl" />
+        <input class="form-control form-control-sm ml-1 rounded " style="padding-right:33px" id="AvatarUrl" placeholder="Avatar URL" @bind="AvatarUrl" @onkeyup="@KeyPressed" />
```
Comment (elmerbool) [inline src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor:58]: Please change to `@this.OnKeyPress`

File: src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor [hunk 2]
```diff
@@ -152,6 +152,15 @@ else
     private Timer refreshTimer;
     private DotNetObjectReference<Tree>? dotNetHelper;
 
+    public async void KeyPressed(KeyboardEventArgs e)
```
Comment (elmerbool) [inline src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor:155]: Rename to `OnKeyPress`

File: src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor [hunk 3]
```diff
@@ -152,6 +152,15 @@ else
...
+    {
+        if(e.Code == "Enter" || e.Code == "NumpadEnter")
+        {
+            if(!string.IsNullOrEmpty(AvatarUrl))
```
Comment (elmerbool) [inline src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor:159]: Rename to `this.AvatarUrl`

File: src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor [hunk 4]
```diff
@@ -152,6 +152,15 @@ else
...
+                await Reload();
```
Comment (elmerbool) [inline src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor:160]: Change to `this.Reload()`

**Review by elmerbool — CHANGES_REQUESTED** [review]
Hi Hans,

Please make the requested changes prior to commit.

File: src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor (hunk 2 above)
Comment (hansnet) [reply src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor]: fixed

File: src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor (hunk 3 above)
Comment (hansnet) [reply src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor]: fixed

File: src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor (hunk 4 above)
Comment (hansnet) [reply src/main/Port.Adapter/UI/Views/Blazor/Pages/Tree.razor]: fixed

**Review by elmerbool — APPROVED** [review]

**Pull request merged**